src/data/reducers/index.js

```diff
@@ -3,9 +3,18 @@ import { combineReducers } from 'redux';
 import assets from './assets';
 import connectionStatus from './connectionStatus';
 
+/* eslint-disable no-undef */
+const courseDetails = () => {
+  if (!courseContext) {
+    throw new Error('Error: Missing Course Context');
```

Comment:
Is this test-able?

Reply:
Will try to get some tests added using the reference that @MichaelRoytman gave! http://redux.js.org/docs/recipes/WritingTests.html